src/views/DandisetLandingView/DandisetUnembargo.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <v-card
+    outlined
+    class="mt-4 px-3"
+  >
+    <v-dialog
+      v-if="showWarningDialog"
+      v-model="showWarningDialog"
+      persistent
+      max-width="60vh"
+    >
+      <v-card class="pb-3">
+        <v-card-title class="text-h5 font-weight-light">
+          Release Dandiset
+        </v-card-title>
+        <v-divider class="my-3" />
+        <v-card-text>
+          <span>
+            This action will unembargo this dandiset. Note that this is a
+          </span>
+          <span class="font-weight-bold">
+            permanent
+          </span>
+          <span>
+            action and is not undoable. Once a dandiset has been released,
+            it will be public forever.
```

Comment:
I'd suggest pulling back from this a bit, with

```suggestion
            it cannot be re-embargoed.
```

instead. "Public forever" is not technically true (admins can help to hide a dandiset that has leaked something illegally, for instance, etc.), and I don't want to scare users into starting to believe something that's not true.

Reply:
https://github.com/dandi/dandiarchive/pull/1021/commits/a9c59551da129aa9c9d26a738b96ed78c6dd0f20